test/widgets/compose_box_test.dart

```diff
@@ -366,6 +370,25 @@ void main() {
       });
     });
 
+    testWidgets('to ChannelNarrow, mandatory topics with empty topic', (tester) async {
```

Comment:
```
compose: Use alternative placeholder if topics are mandatory and missing
```

We have this pair of tests:

- group 'to ChannelNarrow, topics not mandatory'
  - test 'with empty topic'
  - test 'with non-empty topic'

And we have this pair of tests:

- test 'to ChannelNarrow, mandatory topics with empty topic'
- test 'to ChannelNarrow, mandatory topics with non-empty topic'

It seems cleaner if, for both pairs, it either uses `group` or doesn't, with consistent naming.